### PR TITLE
WRP-14788: Update query-string dependency to latest version

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
Enact-DCO-1.0-Signed-off-by: Adrian Cocoara adrian.cocoara@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Updated `query-string` package dependency to latest version
Removed `query-string` dependency from ui/all-samples since it has no usage

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Updated `query-string` package dependency to latest version
Removed `query-string` dependency from ui/all-samples since it has no usage

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
`query-string` v8.1.0 has major breaking changes, but they don't affect our code:
 - `query-string` is now ESM
 - minimum node version > 14.0.0
https://github.com/sindresorhus/query-string/releases/tag/v8.0.0

### Links
[//]: # (Related issues, references)
WRP-14788

### CODE REVIEW

File: ui/all-samples/package.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     "@reduxjs/toolkit": "^1.9.3",
     "ilib": "^14.17.0",
     "prop-types": "^15.8.1",
-    "query-string": "^7.1.3",
+    "query-string": "^8.1.0",
```
Comment (daniel-stoian-lgp) [inline ui/all-samples/package.json:40]: fun fact. i didn't see any usage of this dependency in the ui samples. I think we can remove it

**Review by daniel-stoian-lgp — APPROVED** [review]

**Review by andreitirlalge — APPROVED** [review]
LGTM

File: .travis.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ dist: focal
 language: node_js
 node_js:
     - lts/*
-    - node
+    #- node
```
Comment (hong6316) [inline .travis.yml:5]:
```suggestion
    - "19"
```

**Review by hong6316 — APPROVED** [review]
👍

**Pull request merged**